src/style.css

```diff
@@ -39,11 +39,6 @@ body.tdr table {
   border-collapse: collapse;
 }
 
-body.tdr a {
```

Comment:
Removed this block - the comment from the commit it was made on said that the purpose was to enable typescript conversion, but I think that might've been related to something else in that same pr. This `a` style was overriding the style assigned on the component level to links. Is there a better way to do this? Should we try to keep this?

Reply:
oh yeah, let's keep this. The way that we get links to work is to render a span inside of the link